frontend/packages/ocs-independent-plugin/src/components/dashboard-page/details-card/card.tsx

```diff
@@ -0,0 +1,105 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { referenceForModel, K8sResourceKind } from '@console/internal/module/k8s';
+import { SubscriptionModel } from '@console/operator-lifecycle-manager';
+import {
+  withDashboardResources,
+  DashboardItemProps,
+} from '@console/internal/components/dashboard/with-dashboard-resources';
+import { FirehoseResource, FirehoseResult } from '@console/internal/components/utils';
+import { getName, getInfrastructurePlatform } from '@console/shared';
+import { getOCSVersion } from '@console/ceph-storage-plugin/src/selectors';
+import DashboardCard from '@console/shared/src/components/dashboard/dashboard-card/DashboardCard';
+import DashboardCardTitle from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardTitle';
+import DetailItem from '@console/shared/src/components/dashboard/details-card/DetailItem';
+import DashboardCardBody from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardBody';
+import DashboardCardHeader from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardHeader';
+import { useK8sGet } from '@console/internal/components/utils/k8s-get-hook';
+import { InfrastructureModel } from '@console/internal/models';
+import { ocsVersion } from '@console/ceph-storage-plugin/integration-tests/views/storage-dashboard.view';
```

Comment:
Protractor test related code shouldn't be imported from production React code :smiley: